python/knext/knext/common/cache.py

```diff
@@ -0,0 +1,17 @@
+from cachetools import TTLCache
+
+
+class LinkCache:
+
+    def __init__(self, maxsize: int = 500, ttl: int = 60):
```

Comment:
pre-define global variables for `maxsize` and `ttl`, and we need to get accurate values of these variables through a set of testing to ensure the stability of `fuse result cache`